plugins/node/opentelemetry-instrumentation-graphql/src/instrumentation.ts

```diff
@@ -65,9 +64,7 @@ const DEFAULT_CONFIG: GraphQLInstrumentationConfig = {
 const supportedVersions = ['>=14 <17'];
 
 export class GraphQLInstrumentation extends InstrumentationBase {
-  constructor(
-    config: GraphQLInstrumentationConfig & InstrumentationConfig = {}
```

Comment:
I removed it because `GraphQLInstrumentationConfig` already extends `InstrumentationConfig` so it was redundant